### PR TITLE
Improve CI performance and fix node version

### PR DESCRIPTION
**Pull request by aspirisen**
Discussion https://prezly.slack.com/archives/C029LTE0M/p1651013990025259

### CODE REVIEW

File: .github/workflows/prettier.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Prettier
```
Comment (oleg-semyonov) [inline .github/workflows/prettier.yml:1]: Same comment as in `analytics`: https://github.com/prezly/analytics/pull/2#discussion_r859515418

**Review by oleg-semyonov — CHANGES_REQUESTED** [review]

File: .github/workflows/prettier.yml (hunk 1 above)
Comment (oleg-semyonov) [reply .github/workflows/prettier.yml]: @aspirisen please update the workflow to match [the one in theme-kit-nextjs](https://github.com/prezly/theme-kit-nextjs/blob/main/.github/workflows/lint.yml) (which uses `wearerequired/lint-action`)

**Review by oleg-semyonov — CHANGES_REQUESTED** [review]
Oops, I was a bit to quick to approve..
Please check the Lint workflow checks, it seems that eslint was not even installed on the project 🤔 
The question is if we actually need it here, since there is not much code. 
Maybe running just prettier would be fine, but we can also install `eslint` and `@prezly/eslint-config`. 
Sorry for the confusion :(

**Comment by aspirisen** [comment]
@oleg-semyonov could you look again

**Pull request merged**